tests/cnf/core/network/sriov/tests/qinq.go

```diff
@@ -663,3 +814,130 @@ func disableQinQOnSwitch(switchCredentials *sriovenv.SwitchCredentials, switchIn
 
 	return nil
 }
+
+func defineTestServerPmdCmd(ethPeer, pciAddress string) []string {
+	baseCmd := fmt.Sprintf("dpdk-testpmd -a %s -- --forward-mode txonly --eth-peer=0,%s "+
+		"--cmdline-file=/etc/cmd/cmd_file", pciAddress, ethPeer)
+
+	baseCmd += " --stats-period 5"
+
+	return []string{"/bin/bash", "-c", baseCmd}
+}
+
+func defineTestClientPmdCmd(ethPeer, pciAddress string) []string {
+	baseCmd := fmt.Sprintf(
+		"timeout -s SIGKILL 20 dpdk-testpmd --vdev=virtio_user0,path=/dev/vhost-net,iface=net2 -a %s -- "+
+			"--forward-mode txonly --eth-peer=0,%s", pciAddress, ethPeer)
+
+	baseCmd += " --stats-period 5"
+
+	return []string{baseCmd}
```

Comment:
Do you need "/bin/bash", "-c"?

Reply:
Its on the rxtraffic command. I removed it from there and placed here so it will be the same as the server command.